src/main/java/pl/vm/javaguild/designpatterns/pattern/creational/abstractfactory/model/laptop/Laptop.java

```diff
@@ -0,0 +1,34 @@
+package pl.vm.javaguild.designpatterns.pattern.creational.abstractfactory.model.laptop;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import pl.vm.javaguild.designpatterns.pattern.creational.abstractfactory.model.enums.Producer;
+import pl.vm.javaguild.designpatterns.pattern.creational.abstractfactory.model.graphiccard.GraphicCard;
+import pl.vm.javaguild.designpatterns.pattern.creational.abstractfactory.model.motherboard.Motherboard;
+import pl.vm.javaguild.designpatterns.pattern.creational.abstractfactory.model.processor.Processor;
+
+@Slf4j
+@Setter
```

Comment:
z tego co widze to nigdzie nie uzywamy ani getterow ani setterow w Laptop/LenovoLaptop etc. wiec mozna je usunac